### PR TITLE
Add two topics: Workflow Id and Run Id

### PR DESCRIPTION
**Pull request by djmagee**
## What does this PR do?

Adds two topics:

- What is a Workflow Id? https://deploy-preview-581--mystifying-fermi-1bc096.netlify.app/docs/content/what-is-a-workflow-id/
- What is a Run Id? https://deploy-preview-581--mystifying-fermi-1bc096.netlify.app/docs/content/what-is-a-run-id/

### CODE REVIEW

File: docs/content/what-is-a-run-id.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+id: what-is-a-run-id
+title: What is a Run Id?
+description: A Run Id is a UUID that a Temporal service assigns to each Workflow run.
+tags:
+  - explanation
+---
+
+import RelatedRead from '../components/RelatedRead.js'
+
+A Run Id is a UUID that a Temporal service assigns to each [Workflow](#) run.
```
Comment (flossypurse) [inline docs/content/what-is-a-run-id.md:11]: Per the thread with @dtornow I think we want to do something like this:
> A Run ID is a platform-level identifier for a Workflow Execution that is unique per Namespace.

File: docs/content/what-is-a-run-id.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Temporal guarantees that only one [Workflow Execution](#) with a given [Workflow Id](what-is-a-workflow-id) can be open at a time. But after the [Workflow Execution](#) has completed, if allowed by a configured policy, you might be able to re-execute a [Workflow](#) after it has closed or failed by using the same [Workflow Id](what-is-a-workflow-id).
+- Each such re-execution is called a run. Run Id uniquely identifies a run even if it shares a [Workflow Id](what-is-a-workflow-id) with others.
+
+<RelatedRead
```
Comment (flossypurse) [inline docs/content/what-is-a-run-id.md:16]: Let's use the RelatedReadList component from now on.

File: docs/content/what-is-a-run-id.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+description: A Run Id is a UUID that a Temporal service assigns to each Workflow run.
+tags:
+  - explanation
+---
+
+import RelatedRead from '../components/RelatedRead.js'
+
+A Run Id is a UUID that a Temporal service assigns to each [Workflow](#) run.
+
+- Temporal guarantees that only one [Workflow Execution](#) with a given [Workflow Id](what-is-a-workflow-id) can be open at a time. But after the [Workflow Execution](#) has completed, if allowed by a configured policy, you might be able to re-execute a [Workflow](#) after it has closed or failed by using the same [Workflow Id](what-is-a-workflow-id).
```
Comment (flossypurse) [inline docs/content/what-is-a-run-id.md:13]: Couple things. 
1. Let's lose the bullet points here and paragraph it.
2. Put each sentence on a newline.

File: docs/content/what-is-a-workflow-id.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+id: what-is-a-workflow-id
+title: What is a Workflow Id?
+description: A Workflow Id is a unique identifier for a Workflow Execution.
+tags:
+  - explanation
+---
+
+import RelatedReadList from '../components/RelatedReadList.js'
+
+A Workflow Id is a unique identifier for a [Workflow Execution](#).
```
Comment (flossypurse) [inline docs/content/what-is-a-workflow-id.md:11]: Per the thread with @dtornow I think we want to go with something like:
> A Workflow ID is an application-level identifier for a Workflow Execution that is unique per Namespace.

**Comment by flossypurse** [comment]
I think we will also need to add "What is a Workflow Run?"

File: docs/content/what-is-a-run-id.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+tags:
+  - explanation
+---
+
+import RelatedReadList from '../components/RelatedReadList.js'
+
+A Run Id is a platform-level identifier for a [Workflow Execution](#) that is unique within a [Namespace](docs/server/namespaces).
+
+Temporal guarantees that only one [Workflow Execution](#) with a given [Workflow Id](what-is-a-workflow-id) can be open at a time.
+But after the [Workflow Execution](#) has completed, if allowed by a configured policy, you might be able to re-execute a [Workflow](#) after it has closed or failed by using the same [Workflow Id](what-is-a-workflow-id).
```
Comment (vkarpov15) [inline docs/content/what-is-a-run-id.md:14]: Is there any way to add an example or link to an example of how you could configure a policy that allows re-executing a completed workflow?

**Review by flossypurse — APPROVED** [review]
Lets land this to get it into this format and then iterate from there.

**Pull request merged**